lib/logstash/filters/geoip.rb

```diff
@@ -51,8 +55,8 @@ class LogStash::Filters::GeoIP < LogStash::Filters::Base
   # are included in the event.
   #
   # For the built-in GeoLiteCity database, the following are available:
-  # `city_name`, `continent_code`, `country_code2`, `country_code3`, `country_name`,
-  # `dma_code`, `ip`, `latitude`, `longitude`, `postal_code`, `region_name` and `timezone`.
+  # `city\_name`, `continent\_code`, `country\_code2`, `country\_code3`, `country\_name`,
+  # `dma\_code`, `ip`, `latitude`, `longitude`, `postal\_code`, `region\_name` and `timezone`.
   config :fields, :validate => :array
```

Comment:
actually, ignore this. I would prefer: https://github.com/logstash-plugins/logstash-filter-geoip/pull/64/files#r57260455